packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js

```diff
@@ -339,7 +466,7 @@ moduleFor(
         .then(() => this.assertText('b'));
     }
 
-    ['@test it should update correctly when the controller changes']() {
+    ['@feature(!EMBER_ROUTING_MODEL_ARG) it should update correctly when the controller changes']() {
```

Comment:
(gonna stop commenting on these now, it seems the remainder of the file will need the same general tweaks/updates that I mentioned from above)